Using backticks in the template causes an "invalid_arguments" error

## Orb version: 4.4.2

## What happened:
When using backticks in the template (e.g. for markdown messages), they are not escaped (when providing an inline custom message, they are).
This causes the evaluation to fail (with a `-bash: content-inside-backticks: command not found`, which in turn leaved the rendered template (`T2`) empty, getting the following response from Slack:
```json
{"ok":false,"error":"invalid_arguments","response_metadata":{"messages":["[ERROR] missing required field: channel"]}}
```

## Expected behavior:
Escaping the backticks too.

## Additional Information:
`\\` is also escpaed for inline custom messages but not for templates.

--- a/src/scripts/notify.sh
+++ b/src/scripts/notify.sh
@@ -10,6 +10,7 @@ BuildMessageBody() {
         T2=$(eval echo \""$CUSTOM_BODY_MODIFIED"\")
     elif [ -n "${SLACK_PARAM_TEMPLATE:-}" ]; then
         TEMPLATE="\$$SLACK_PARAM_TEMPLATE"
+        # shellcheck disable=SC2016
         T1=$(eval echo "$TEMPLATE" | sed 's/\\/\\\\/g' | sed 's/"/\\"/g' | sed 's/`/\\`/g')
         T2=$(eval echo \""$T1"\")
     else

--- a/src/scripts/notify.sh
+++ b/src/scripts/notify.sh
@@ -10,7 +10,7 @@ BuildMessageBody() {
         T2=$(eval echo \""$CUSTOM_BODY_MODIFIED"\")
     elif [ -n "${SLACK_PARAM_TEMPLATE:-}" ]; then
         TEMPLATE="\$$SLACK_PARAM_TEMPLATE"
-        T1=$(eval echo "$TEMPLATE" | sed 's/"/\\"/g')
+        T1=$(eval echo "$TEMPLATE" | sed 's/\\/\\\\/g' | sed 's/"/\\"/g' | sed 's/`/\\`/g')
         T2=$(eval echo \""$T1"\")
     else
         echo "Error: No message template selected."

